docs/about.md

```diff
@@ -0,0 +1,70 @@
+# Cloud automation for Key Protect
```

Comment:
Should not be key protect, maybe `Cloud automation for account configuration`?

Reply:
may bad copy paste issues